src/js/eva-config.js

```diff
@@ -75,6 +75,20 @@ function deDuplicatedSpeciesList(speciesList) {
     });
 }
 
+function deDuplicateSpeciesListWithAssemblyAccession(speciesList) {
+    //In the case of multiple assemblies like Grch37 and Grch37.p13 in the species list, use only the first one since
+    //they both have the same assembly name and variants from either assembly reside within the same database
+    speciesList = _.sortBy(speciesList, function(listItem) {
+                                            return [listItem.assemblyAccession,
+                                                    listItem.taxonomyEvaName.toLowerCase()].join(); });
+    return _.uniq(speciesList, function(listItem, key, unused) {
+        // Different species can have assembly codes which coincide with each other, e. g. "Pepper Zunla 1 Ref_v1.0"
+        // (taxonomy ID 4072) and L_crocea_1.0 (taxonomy ID 215358) both have the assembly code of "10". Because of
+        // this, we need to have uniq over a tuple of taxonomy ID and assembly code.
+        return [listItem.taxonomyId, listItem.assemblyAccession].join();
```

Comment:
wait, is this the assembly code or assembly accession? 
If it's the assembly accession, then
- from other parts of the code, my understanding is that the comments are outdated, is that right?
- the function is called deduplication, but what are you deduplicating if the assembly accessions are always different? the only exception is an assembly shared across species (like sheep/goat) but in that case it won't be deduplicated either because you take into account the taxonomy. I don't understand the purpose of this function. I thought it was meant to show in the dropdown only "GRCh37" instead of "GRCh37", "GRCh38.p3" and "GRCh38.p12".

if it's the assembly code (as the comments suggest), then
- I wouldn't call the assembly code "assembly accession", they are not the same. I guess it's ok too if you put the general term "assembly".